src/lib/Preview.js

```diff
@@ -467,6 +461,33 @@ class Preview extends EventEmitter {
         });
     }
 
+    /**
+     * Fetch file info
```

Comment:
How about `Fetch and return file info, which can then be cached or stored externally`? We also like adding a line break after the description before the first @ param